standalone/standalone.py

```diff
@@ -984,37 +1032,26 @@ def get_vol() -> list[kubernetes.client.V1Volume]:
     ]
 
 
-def create_sdg_job(
-    namespace: str,
-    job_name: str,
-    num_instructions_to_generate: int,
-    exec_git_clone_op_repo_url: str = "",
-    exec_git_clone_op_repo_branch: str = "",
-    exec_git_clone_op_repo_pr: str = "",
-) -> kubernetes.client.V1Job:
+def create_sdg_container(
+    sdg_serving_model_secret: str,
+    num_instructions_to_generate: int = 30,
+    exec_git_clone_op_repo_branch: str = None,
+    exec_git_clone_op_repo_pr: str = None,
```

Comment:
these look like they are not used

Reply:
They are arguments to the `sdg_op` function. `exec_git_clone_op_repo_branch` and `exec_git_clone_op_repo_pr` are just not exposed to the CLI. I made a comment about that. Thanks!